### PR TITLE
Refactor biomedImageTable class and tests.

### PR DESCRIPTION
**Pull request by cshruti87**

### CODE REVIEW

**Review by mumuMia — COMMENTED** [review]
Good job, Shruti. I only have a few minor comments.

File: cvpy/image/Image.py [hunk 1]
```diff
@@ -190,91 +190,6 @@ def convert_to_CAS_column(s):
         s = str.replace(str.replace(s, '{', '_'), '}', '_')
```
[the remark this answers is not in the document]
Comment (cshruti87) [reply cvpy/image/Image.py:190]: Sure, I will delete that file.

File: cvpy/biomedimage/BiomedImageTable.py [hunk 2]
```diff
@@ -1,33 +1,316 @@
...
 
-import numpy as np
+from typing import Dict, List
+import struct
+import numpy
+from swat import CASTable
 from cvpy.base.ImageTable import ImageTable
 from cvpy.biomedimage.LabelConnectivity import LabelConnectivity
+from cvpy.utils.RandomNameGenerator import RandomNameGenerator
+from cvpy.image.Image import Image
```
[the remark this answers is not in the document]
Comment (cshruti87) [reply cvpy/biomedimage/BiomedImageTable.py:28]: NaturalImageTable is not used anywhere in this file, so is it for future reference?

**Review by ptelang — COMMENTED** [review]
Hi Shruti, Thanks for working on this change. Please see my comments.

File: cvpy/tests/test_biomed_imagetable.py [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+        # Load the image
+        image = self.s.CASTable('image', replace=True)
+        self.s.image.loadImages(path='biomedimg/simple.png',
+                                casOut=dict(name='image', replace='TRUE'),
+                                addColumns={"WIDTH", "HEIGHT", "DEPTH", "CHANNELTYPE", "SPACING"},
+                                caslib='dlib',
+                                decode=True)
+
+        # Construct Biomed object
+        biomed = BiomedImageTable(table=image)
```
[the remark this answers is not in the document]
Comment (cshruti87) [reply cvpy/tests/test_biomed_imagetable.py:56]: Done

**Comment by cshruti87** [comment]
Hi Pankaj, I have addressed your comments. Please take a look.

**Review by ptelang — APPROVED** [review]
Thanks for addressing all of the comments Shruti.

**Pull request merged**